webrender/src/renderer.rs

```diff
@@ -3828,6 +3830,7 @@ pub trait OutputImageHandler {
 pub trait ThreadListener {
     fn thread_started(&self, thread_name: &str);
     fn thread_stopped(&self, thread_name: &str);
+    fn new_render_backend_thread(&self, renderer_id: Option<u64>);
```

Comment:
How are you going to store the state related to the APZ logic on the render backend? wouldn't it make sense to pass an object in the renderer options with an init method that runs on the render backend before starting the loop and that is called by the render backend at the right times?

Reply:
I don't think we'll actually need to store any state in the render backend. We'll just need this call plus one other at the time of the scene swap. I'm doing this incrementally so I just wanted to land this for now as I'll need it for bug 1447299, and then when I need the other callback we can introduce a new object at that point if it makes sense.